C/CoolProp/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "CoolProp"
+version = v"6.4.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://sourceforge.net/projects/coolprop/files/CoolProp/$version/source/CoolProp_sources.zip", "b10b2be2f88675b7e46cae653880be93558009c8970d23e50ea917ce095791f6"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+
+sed -i 's/Windows/windows/' CoolProp.sources/dev/Tickets/60.cpp
+sed -i 's/Windows/windows/' CoolProp.sources/src/CPfilepaths.cpp
+sed -i 's/.*-m.*BITNESS.*//' CoolProp.sources/CMakeLists.txt
+
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_FIND_ROOT_PATH=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DCOOLPROP_SHARED_LIBRARY=ON ../CoolProp*/
+VERBOSE=ON cmake --build . --config Release --target CoolProp -- -j${nproc}
+mkdir -p $prefix/lib
```

Comment:
Better to use `${libdir}`. Can't you run `make install`?

Reply:
Make install puts in per-architecture subdirectories of lib, I'll use `${libdir}`